src/raiden_libs/register_service.py

```diff
@@ -0,0 +1,145 @@
+import sys
+from typing import Dict
+
+import click
+import structlog
+from web3 import Web3
+from web3.contract import Contract, ContractFunction
+from web3.middleware import construct_sign_and_send_raw_middleware
+
+from raiden.utils.typing import Address, BlockNumber, TokenAmount
+from raiden_contracts.constants import (
+    CONTRACT_CUSTOM_TOKEN,
+    CONTRACT_SERVICE_REGISTRY,
+    CONTRACT_USER_DEPOSIT,
+)
+from raiden_libs.cli import blockchain_options, common_options
+from raiden_libs.contract_info import CONTRACT_MANAGER
+from raiden_libs.utils import private_key_to_address
+
+log = structlog.get_logger(__name__)
+
+
+@blockchain_options(contracts=[CONTRACT_SERVICE_REGISTRY, CONTRACT_USER_DEPOSIT])
+@click.command()
+@click.option(
+    "--deposit",
+    default=100 * 10 ** 18,  # 100 RDN
+    type=click.IntRange(min=1),
+    help="Amount of tokens to deposit in the ServiceRegistry",
+)
+@click.option("--service-url", type=str, help="URL for the services to register")
+@common_options("register_service")
+def main(
+    private_key: str,
+    state_db: str,  # pylint: disable=unused-argument
+    web3: Web3,
+    contracts: Dict[str, Contract],
+    start_block: BlockNumber,  # pylint: disable=unused-argument
+    deposit: TokenAmount,
+    service_url: str,
+) -> None:
+    """
+    Registers the address of a service deployment with the `ServiceRegistry`.
+
+    It also set or updates the URL of the services deployment.
```

Comment:
```suggestion
    It also sets or updates the URL of the services deployment.
```

Maybe add one more sentence to say that the `private_key`s address will be registered. Or is that obvious?